pyproject.toml

```diff
@@ -78,6 +77,15 @@ skip_covered = true
 
 [tool.ruff]
 src = ["hijack", "tests"]
+line-length = 88
+indent-width = 4
+
```

Comment:
I'd recommend to specify the minimum python version we are supporting in
target-version

Reply:
https://docs.astral.sh/ruff/settings/#target-version

Reply:
Oh. I see. It's not needed. Because ruff respect requires-python